ui/js/controllers/perf/dashboard.js

```diff
@@ -25,7 +23,8 @@ perf.controller('dashCtrl', [
                 try {
                     $scope[k] = phDashboardValues[$scope.topic][k];
                 } catch (TypeError) {
-                    $log.error('"'+ k + '" option not found in "' + $scope.topic + '" dashboard config');
+                    // eslint-disable-next-line no-console
+                    console.error('"'+ k + '" option not found in "' + $scope.topic + '" dashboard config');
```

Comment:
I know this is pre-existing, but perhaps it's worth switching this to a template literal?

Reply:
ahh right, of course.  Good call...  :)